app/assets/javascripts/discourse/app/templates/user-invited-show.hbs

```diff
@@ -90,50 +73,63 @@
+                  <th>{{i18n "user.invited.invited_via"}}</th>
+                  <th>{{i18n "user.invited.groups"}}</th>
+                  <th>{{i18n "user.invited.sent"}}</th>
+                  <th>{{i18n "user.invited.expires_at"}}</th>
+                  <th></th>
+                </tr>
+              </thead>
+              <tbody>
+                {{#each model.invites as |invite|}}
+                  <tr>
                     <td>
-                      {{#if invite.expired}}
-                        <div>{{i18n "user.invited.expired"}}</div>
-                      {{/if}}
-                      {{#if invite.rescinded}}
-                        {{i18n "user.invited.rescinded"}}
+                      {{#if invite.email}}
+                        {{invite.email}}
                       {{else}}
-                        {{d-button icon="times" action=(action "rescind") actionParam=invite label="user.invited.rescind"}}
+                        {{i18n "user.invited.invited_via_link" count=invite.redemption_count max=invite.max_redemptions_allowed}}
                       {{/if}}
                     </td>
                     <td>
-                      {{#if invite.reinvited}}
-                        <div>{{i18n "user.invited.reinvited"}}</div>
+                      {{#each invite.groups as |g|}}
+                        <a href="/g/{{g.name}}">{{g.name}}</a>
                       {{else}}
-                        {{d-button icon="sync" action=(action "reinvite") actionParam=invite label="user.invited.reinvite"}}
-                      {{/if}}
+                        &mdash;
+                      {{/each}}
                     </td>
-                  {{else if inviteLinks}}
-                    <td>{{d-button icon="link" action=(route-action "editInvite" invite.invite_key) label="user.invited.copy_link"}}</td>
-                    <td>{{format-date invite.created_at}}</td>
-                    <td>{{number invite.redemption_count}} / {{number invite.max_redemptions_allowed}}</td>
-                    <td colspan="2">{{ invite.group_names }}</td>
-                    <td>{{raw-date invite.expires_at leaveAgo="true"}}</td>
+                    <td>{{format-date invite.updated_at}}</td>
                     <td>
-                      {{#if invite.rescinded}}
-                        {{i18n "user.invited.rescinded"}}
+                      {{#if invite.expired}}
+                        {{i18n "user.invited.expired"}}
                       {{else}}
-                        {{d-button icon="times" action=(action "rescind") actionParam=invite label="user.invited.rescind"}}
+                        {{raw-date invite.expires_at}}
                       {{/if}}
                     </td>
-                  {{/if}}
-                </tr>
-              {{/each}}
-            </tbody>
-          </table>
-          {{conditional-loading-spinner condition=invitesLoading}}
+                    <td style="white-space: nowrap; width: 100px;">
```

Comment:
Do we have to use `style`? Using scss is preferred.

Reply:
Sorry, I missed this.